Doc/whatsnew/3.10.rst

```diff
@@ -186,6 +186,11 @@ by :func:`curses.color_content`, :func:`curses.init_color`,
 support is provided by the underlying ncurses library.
 (Contributed by Jeffrey Kintscher and Hans Petter Jansson in :issue:`36982`.)
 
+encodings
+---------
+:func:`encodings.normalize_encoding` now ignores non-ASCII letters.
```

Comment:
Oh wait, in Unicode, a "letter" has a special meaning. Please replace "non-ASCII letters" with "non-ASCII characters". Here, but also in the NEWS entry and in the test (everywhere).

Reply:
ok, updated.